Cargo.toml

```diff
@@ -0,0 +1,36 @@
+[workspace]
+resolver = "2"
+
+# Smart contracts currently has to be standalone crates and not part of the repo workspace
+# due to how cargo-concordium verifiable build works. Hence, smart contract crates in teh repo
+# are not members of the workspace.
+members = [
+   "chain-prometheus-exporter",
+   "generator",
+   "genesis-creator",
+   "id-verifier",
+   "kpi-tracker",
+   "notification-server",
+   "recover-id-object",
+   "state-compare",
+   "wallet-proxy-load-simulator",
+   "company-id-gui/src-tauri"
+]
+
+# The crates in the concordium-rust-sdk submodule are implicitly members
+# of the workspace since they are local path dependencies of the explicit members.
+# Hence, we list the explicit members here as default members such that cargo commands
+# on these by default.
+default-members = [
+   "chain-prometheus-exporter",
+   "generator",
+   "genesis-creator",
+   "id-verifier",
+   "kpi-tracker",
+   "notification-server",
+   "recover-id-object",
+   "state-compare",
+   "wallet-proxy-load-simulator",
+   # The package in company-id-gui/src-tauri is currently not among the default members, since it requires additinal tools to build, but there should be nothing wrong with adding it, if the build works
+#   "company-id-gui/src-tauri"
```

Comment:
IMO, we should add it and make sure CI has the tools available to build it. Otherwise we might have a situation where errors are not caught?